src/DroneSafetyApi/Services/HazardsToHeatmapResponse.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using DroneSafetyApi.Models;
+using Microsoft.Azure.Documents.Spatial;
+
+namespace DroneSafetyApi.Services
+{
+    public class HazardsToHeatmapResponse : HazardsToHeatmapsResponseNoCompositionSendAllSources
+    {
+
+        public override HeatMap ConvertToHeatmap(BoundingBox area, int width, int height, IEnumerable<Hazard> hazards)
+        {
+            // Initialise Heatmap
+            HeatMap heatmap = new HeatMap(area.Min.Longitude, area.Max.Longitude, area.Min.Latitude, area.Max.Latitude, width, height);
+
+            foreach (Hazard hazard in hazards)
```

Comment:
Sure, but we are also doing a case split on the type of shape, right? So if we had three separate lists, we wouldn't have this ugly case statement.